### ISSUE TITLE
Use "pure" specifier in Math functions

### ISSUE THREAD

**Issue by HaleTom** [OpenZeppelin/zeppelin-solidity#481]
I came to your repository via [this page](https://medium.com/blockchannel/state-specifiers-and-staticcall-d50d5b2e4920) which illustrates the effect of using the Solidity 0.4.16 compiler on this code.

Would you consider using the [`pure`](https://github.com/ethereum/solidity/pull/2745) specifier to avoid these warnings?

**Comment by hbdgr** [comment]
Have same feelings. I even created PR #480 fixing this.
Unfortunately, it does not pass Travis tests.
```
/home/travis/build/OpenZeppelin/zeppelin-solidity/coverageEnv/contracts/math/SafeMath.sol:15:47: ParserError: Expected token LBrace got reserved keyword 'Pure'
```
looks strange.

**Comment by Qqwy** [comment]
Seems to me like Travis is using an older compiler version, since the keyword 'pure' is not recognized.

**Comment by frangio** [comment]
Thanks for reporting everyone. Fixed in #573.
